src/app/cli/src/commands/verify_command.rs

```diff
@@ -46,7 +46,7 @@ impl VerifyCommand {
         I: Iterator<Item = DatasetRef>,
     {
         Self {
-            local_repo,
+            dataset_repo,
```

Comment:
In future whenever you think we need to mass-rename something let's do it on `master` branch as a quick isolated change - this would make this PR a lot easier to read.